exercises/practice/beer-song/.meta/example.py

```diff
@@ -1,19 +1,19 @@
 def recite(start, take=1):
     results = []
-    for i in range(start, start - take, -1):
-        results.extend(verse(i))
-        if i > start - take + 1:
-            results.append('')
+    for idx in range(start, start - take, -1):
+        results.extend(verse(idx))
+        if idx > start - take + 1:
+            results.append("")
     return results
 
 
 def verse(number):
     return [
-        ''.join([
+        "".join([
             "{} of beer on the wall, ".format(_bottles(number).capitalize()),
             "{} of beer.".format(_bottles(number))
         ]),
-        ''.join([
+        "".join([
             _action(number),
             _next_bottle(number)
         ])
```

Comment:
Whenever `"".join()` is being used, it's probably better to use a generator expression over a list, since a list creates data in memory.  I can't suggest due to formatting, but I think both of these should omit the list.